### DISCUSSION TITLE
Missing Data

### DISCUSSION THREAD

**Discussion by seeohsee** [ranaroussi/yfinance#1884] [Yahoo's data]
This appears to be a Yahoo Finance issue, and not a yfinance one. If so, feel free to close this issue...

The S&P 500 index (^GSPC) appears to be missing daily data. One example I just came across is on Monday, 10/29/2012 and Tuesday, 10/30/2012.
<img width="893" alt="Screen Shot 2020-12-05 at 3 19 48 PM" src="https://user-images.githubusercontent.com/63052895/101262605-110bfc00-370e-11eb-99d9-d6b4a433c9f7.png">

I verified that the markets were open during those two days (see https://holidaystracker.com/stock-market/new-york-stock-exchange-holidays-2012/), but for some reason Yahoo doesn't have any historical data for those trading days.

Has anyone else come across missing daily (or weekly) historical data?

**Comment by cyrbaufr** [comment]
I also noticed a lot of missing historical data on numerous world stockmarket indices. This is really strange. Sometimes data are available for the previous session. The next day they disappear. Two days later they are back. This also happens for equities and currencies. This is annoying because it triggers bugs in my python scripts. All errors are probably coming from Yahoo Finance.
When US markets are closed it also seems that they do not fetch data on other world stockmarket indices.
